litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -589,7 +589,7 @@ spec:
           volumeMounts:
             - mountPath: /tmp/gitops
               name: gitops-storage
-            - mountPath: /data/tmp
+            - mountPath: /tmp/version
```

Comment:
probably `/tmp/hub` would be a better name

Reply:
@gdsoumya We are planning to add a separate env for chaoshub paths. Will take this in the refactoring PR.